src/ios/CDVCapture.m

```diff
@@ -266,7 +267,18 @@ - (void)captureVideo:(CDVInvokedUrlCommand*)command
         // iOS 4.0
         if ([pickerController respondsToSelector:@selector(cameraCaptureMode)]) {
             pickerController.cameraCaptureMode = UIImagePickerControllerCameraCaptureModeVideo;
-            // pickerController.videoQuality = UIImagePickerControllerQualityTypeHigh;
+            switch ((int) (quality ? [quality doubleValue] * 10 : -1)) {
+                case 0:
+                    pickerController.videoQuality = UIImagePickerControllerQualityTypeLow;
+                    break;
+                case 5:
+                default:
```

Comment:
Change `default` to `high` to match adding back `this.quality = 1;`

Reply:
I moved default to `case 10`, for `UIImagePickerControllerQualityTypeHigh`, corresponding to `quality = 1`.